src/main/java/com/flightstats/hub/config/WebhookProperties.java

```diff
@@ -0,0 +1,38 @@
+package com.flightstats.hub.config;
+
+import javax.inject.Inject;
+
+public class WebhookProperties {
+
+    private final PropertiesLoader propertiesLoader;
+
+    @Inject
+    public WebhookProperties(PropertiesLoader propertiesLoader) {
+        this.propertiesLoader = propertiesLoader;
+    }
+
+    public int getCallbackTimeoutMin() {
```

Comment:
I think this name is somewhat ambiguous, when I see "Min" I think minutes not minimum, I would prefer something like "getCallbackTimeoutMinimumInSec" and think it would be consistent to add the time unit to the max get method that follows

Reply:
I might add a task somewhere (that'll likely be deprioritized) to use Duration types for properties that have time units coded into their name. sot this'd end up being `Duration getCallbackTmeoutMinimum`. I've found that using the right object type for the value generally DRYs up a lot of the places that end up doing time-based calculations with the method.